website/blog/2018-06-27-supporting-jest-open-source.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Supporting Jest Open Source
+author: Rick Hanlon II
+authorURL: https://twitter.com/rickhanlonii
+authorFBID: 160800566
+---
+
+Jest is maintained by a combination of Facebook employees and a community of open source contributors.
+
+Our open source contributors work on behalf of the community to create new features, fix bugs, and maintain the issue backlog. Facebook employees do the same, focused on the problems specific to Facebook's scale, and run all proposed releases on Facebook's enormous test suite before publishing.
+
+All of us do our part to make testing delightful.
+
+At the Jest Summit we announced the Jest Open Collective. This is our way to help support the efforts of the open source Jest community. **To be absolutely clear: Facebook employees cannot benefit from the Jest Open Collective.** The collective's sole purpose is to support the open source side of the Jest community.
+
+In this post we'll outline what the Jest Open Collective is, the structure, and the goals we have.
+
+# The Jest Open Collective
+![Banner image for Jest Open Collective](/img/blog/collective.png)
+
+Open Collective is a platform to manage groups of people transparently. Jest joins projects like [Webpack](https://opencollective.com/webpack), [Babel](https://opencollective.com/babel), [Mocha](https://opencollective.com/mochajs), [Preact](https://opencollective.com/preact#budget), [Vue](https://opencollective.com/vuejs), and many more in supporting the open source community through the Open Collective platform.
+
+## What is the Jest Open Collective
+
+The Jest Open Collective is a group of open source contributors who operate in full transparency to:
+
+* Receive funds from backers and sponsors
+* Approve expenses submitted by the community
+* Give everyone visibility on the budget
+
+## What is the structure of the collective
+
+The Jest Open Collective is currently managed by three non-Facebook core contributors from the open source community:
+
+- [Michal Pierzchala](https://twitter.com/thymikee)
+- [Simen Bukkhus](https://github.com/simenb)
+- [Rick Hanlon](https://twitter.com/rickhanlonii)
+
+Michal has been an active Jest contributor since September 2016, Simen joined the core team in June 2017, and Rick joined in January. They're responsible for setting the budget, approving expenses, and managing admins of the collective. We have elected to keep Christoph Nakazawa as an adviser.
+
+There are two levels of support for the collective: Backers and Sponsors.
+
+### Backers
+Backers of the collective are individuals contributing at least $2/month. We'll include a list of backers on the Jest homepage, README on github/yarn/npm, and Contributors page.
```

Comment:
We should send these folks stickers if they keep backing Jest for a year or so. The only issue I have with mailing stickers is normally the enormous amount of manual labor. Maybe we can distribute it between a couple of us.